sysutils/screen/Portfile

```diff
@@ -53,12 +53,9 @@ post-extract {
 }
 
 use_autoconf    yes
-autoconf.cmd    ./autogen.sh
 
 # need for autoconf
 depends_build-append \
-    port:autoconf    \
-    port:automake    \
     port:libtool
```

Comment:
see the warning: `Warning: Dependency port:libtool specified multiple times in depends_build`

Reply:
@dgilman can you make the last change so that the PR can be merged?